code/game/objects/effects/snowcloud.dm

```diff
@@ -125,12 +125,15 @@
 /obj/item/snowball
 	name = "snowball"
 	desc = "Get ready for a snowball fight!"
-	throwforce = 10
 	icon_state = "snowball"
-	damtype = STAMINA
+	/// The amount of stamina damage to do on hit.
+	var/stamina_damage = 10
 
 /obj/item/snowball/throw_impact(atom/target)
 	..()
```

Comment:
Doesn't this also mean that it will hit a person even if he blocks it?

Reply:
Correct, fixed